server_tests/test/clientTests.js

```diff
@@ -799,38 +798,37 @@ describe("RUST API TEST SUITE", function() {
                 if (isActive) {
                     return testClient.closeSession(sessionId)
                     .then( function onSuccess (res) {
-                        Comparison.oscCloseSessionOutput(res);
+                        validate.done(res.body, schema.names.commandCloseSession);
                     }, wrapError);
                 }
             }, wrapError);
         });
 
         it("Expect success. camera.getOptions gets correct options when gettable options are set to supported values", function() {
             return testClient.getOptions(sessionId, specifiedOptions)
-            .then( function onSuccess (res) {
-                Comparison.oscGetOptionsOutput(specifiedOptions, res);
-            }, wrapError);
+            .then( (res) => validate.done(res.body, schema.names.commandGetOptions),
```

Comment:
No, the http statuses are already implicitly verified by the library. If the promise succeeds, it means the status was 200.